dotnet/Makefile

```
@@ -123,7 +121,8 @@ Microsoft.NET.Workload.$(1)/LICENSE: $(TOP)/LICENSE
 WORKLOAD_TARGETS += Microsoft.NET.Workload.$(1)/WorkloadManifest.json
 WORKLOAD_TARGETS += Microsoft.NET.Workload.$(1)/LICENSE
 endef
-$(foreach platform,$(DOTNET_PLATFORMS),$(eval $(call WorkloadTargets,$(platform),$(shell echo $(platform) | tr A-Z a-z),$($(platform)_NUGET_VERSION_NO_METADATA))))
+$(foreach platform,$(DOTNET_PLATFORMS),$(eval $(call WorkloadTargets,$(platform),$(shell echo $(platform) | tr A-Z a-z),$($(platform)_NUGET_VERSION_NO_METADATA), \
+	$(if $(findstring $(platform),$(DOTNET_WINDOWS_PLATFORMS)),targets/WorkloadManifest.windows.template.json,targets/WorkloadManifest.template.json))))
```

I think this will stop generating workloads for platforms included in DOTNET_WINDOWS_PLATFORMS, are you sure you don't want to add a line instead:

```suggestion
$(foreach platform,$(DOTNET_PLATFORMS),$(eval $(call WorkloadTargets,$(platform),$(shell echo $(platform) | tr A-Z a-z),$($(platform)_NUGET_VERSION_NO_METADATA,targets/WorkloadManifest.template.json))))
$(foreach platform,$(DOTNET_WINDOWS_PLATFORMS),$(eval $(call WorkloadTargets,$(platform),$(shell echo $(platform) | tr A-Z a-z),$($(platform)_NUGET_VERSION_NO_METADATA,targets/WorkloadManifest.windows.template.json))))
```

---

I want to use the new "windows" template file for all workloads that include a windows SDK.  These changes _should_ result in the `Microsoft.NET.Workload.iOS` workload declaring this new Windows only pack, however since it's conditionally imported in `WorkloadManifest.targets` it will never need to be installed on macOS.  This all seemed to work locally but it looks like CI didn't run for this yesterday so I'm still waiting to check an official build.  I don't want to introduce new Windows specific workloads (just one new workload pack).  We should be able to continue to ship the same `Microsoft.NET.Workload.iOS` workload on both macOS and Windows.